yb-voyager/src/srcdb/srcdb.go

```diff
@@ -51,6 +51,7 @@ type SourceDB interface {
 	GetTableToUniqueKeyColumnsMap(tableList []*sqlname.SourceName) (map[string][]string, error)
 	ClearMigrationState(migrationUUID uuid.UUID, exportDir string) error
 	GetNonPKTables() ([]string, error)
+	GetTablesWithReplicaIdentityNotFull(tableList []*sqlname.SourceName) ([]string, error)
```

Comment:
We shouldn't have a func with this name in the SourceDB interface, because "ReplicaIdentity" is PG-specific. I see two options:
1. Remove it from the interface; Have the func `checkIfTablesHaveReplicationIdentityFull` use the concrete PG struct by using [type assertion](https://go.dev/tour/methods/15)
2.  Declare a generic method like `ValidateTablesReadyForLiveMigration` in the interface, and return a generic set of validation results for a set of tables. (map[string]string[])

Reply:
Chose the 2nd option since it was easier to implement.

Reply:
We can add further checks in the future if necessary in the same function